ecommerce_worker/utils.py

```diff
@@ -46,19 +48,36 @@ def get_configuration(variable, site_code=None):
     return setting_value
 
 
-def get_ecommerce_client(url_postfix='', site_code=None):
+def get_access_token():
     """
-    Get client for fetching data from ecommerce API.
-    Arguments:
-        site_code (str): (Optional) The SITE_OVERRIDES key to inspect for site-specific values
-        url_postfix (str): (Optional) The URL postfix value to append to the ECOMMERCE_API_ROOT value.
+    Returns an access token for this site's service user.
```

Comment:
Please pay attention to the multiple-docstrings recommended format, check the OeX python style guide for more details.

Reply:
fixed